tests/test_terminal.py

```diff
@@ -276,4 +276,10 @@ def test_shell_command_override(
     pytest.importorskip("traitlets", minversion=min_traitlets)
     argv = shlex.split(f"--ServerApp.terminado_settings={terminado_settings}")
     app = jp_configurable_serverapp(argv=argv)
-    assert app.web_app.settings["terminal_manager"].shell_command == expected_shell
+    if os.name == "nt":
+        assert app.web_app.settings["terminal_manager"].shell_command in (
+            expected_shell,
+            " ".join(expected_shell),
+        )
```

Comment:
Not sure if this is acceptable or not, but it does describe the current behavior on Windows...

Reply:
Well but then this test would still fail. So yes, I think that in any case harmonising this to comparison of strings in tests is fine.